diff --git a/es-core/src/Renderer_init_sdlgl.cpp b/es-core/src/Renderer_init_sdlgl.cpp
index 00175f3..cab211a 100644
--- a/es-core/src/Renderer_init_sdlgl.cpp
+++ b/es-core/src/Renderer_init_sdlgl.cpp
@@ -36,6 +36,9 @@ namespace Renderer
 			return false;
 		}
 
+		//hide mouse cursor
+		initialCursorState = SDL_ShowCursor(0) == 1;
+
 		SDL_GL_SetAttribute(SDL_GL_RED_SIZE, 8);
 		SDL_GL_SetAttribute(SDL_GL_GREEN_SIZE, 8);
 		SDL_GL_SetAttribute(SDL_GL_BLUE_SIZE, 8);
@@ -108,9 +111,6 @@ namespace Renderer
 				LOG(LogWarning) << "Tried to enable vsync, but failed! (" << SDL_GetError() << ")";
 		}
 
-		//hide mouse cursor
-		initialCursorState = SDL_ShowCursor(0) == 1;
-
 		return true;
 	}
